src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsModal.component.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import 'vs/css!./modal';
+import { Component, Input } from '@angular/core';
+import { calloutDialogBodyBackground, calloutDialogForeground, calloutDialogInteriorBorder } from 'sql/platform/theme/common/colorRegistry';
+import { IColorTheme, ICssStyleCollector, registerThemingParticipant } from 'vs/platform/theme/common/themeService';
+
+@Component({
+	selector: 'notebook-views-modal-component',
```

Comment:
I'm still not sure why this is a "modal" component. It's not clear from your gif what part this is actually corresponding to - could you include a screenshot here for what this is?

Reply:
This is what it looks like. It just displays the input for the cell, covering the notebook. Right now it looks very basic as we are waiting for the design team to provide a design, but we needed a solution for now.

![image](https://user-images.githubusercontent.com/8183814/128563590-b5068a66-5890-4999-ac8b-c466a7a0a5a1.png)